src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -2775,13 +2774,12 @@ private BoundExpression BindIsOperator(BinaryExpressionSyntax node, DiagnosticBa
                     operand = ToBadExpression(operand);
                 }
 
-                var boundConstantPattern = BindConstantPattern(
-                    node.Right, operand.Type, node.Right, node.Right.HasErrors, isPatternDiagnostics, out wasExpression);
-                boundConstantPattern.WasCompilerGenerated = true;
```

Comment:
>boundConstantPattern.WasCompilerGenerated = true; [](start = 16, length = 49)

Should this stay (in the scenario where we don't enter the `if` with a `return`)?

Reply:
This variable does not exist any longer.  If we don't enter the if and return then we don't have a constant pattern.

---
In reply to: [362987700](https://github.com/dotnet/roslyn/pull/40503#discussion_r362987700) [](ancestors = 362987700)